thavalon-server/src/game/role.rs

```diff
@@ -32,14 +60,22 @@ impl Role {
         !self.is_good()
     }
 
+    pub fn team(self) -> Team {
+        if self.is_good() {
+            Team::Good
+        } else {
```

Comment:
nit: Don't need else here, can just return Team::Evil if the other condition is false (assuming that Rust allows that)

Reply:
It would, but then the `Team::Good` line would need a return`